src/webprojects/relay.md

```diff
@@ -0,0 +1,80 @@
+# Firefox Relay: how to localize and test
+
+Firefox Relay⁩ email masks protect your true email address from public view, automatically forwarding messages to your true inbox. Now you can receive only the messages you want in your inbox. For Canada and the US, phone mask feature as a premium service is offered.
+
+Firefox Relay consists of two parts:
+* [Firefox Relay Website](https://pontoon.mozilla.org/projects/firefox-relay-website)
+* [Firefox Relay Add-on](https://pontoon.mozilla.org/projects/firefox-relay-add-on)
+
+## Firefox Relay Website
+
+This project is open to localization in all languages. The free version with a limit of 5 email masks is available to all users. However, Relay Premium, a paid service, is available in [these markets](https://docs.google.com/document/d/1ww1ab5Vhr4AISMlWZvTe0cnz8GD16F8xE5vegcJZNwQ/edit#) at the moment. The team will continue market expansion as well as adding the product features and making improvements.
+
+Phone masking, a paid service, is offered to limited markets. Localizable content is limited in languages where the markets are offered the Relay Premium services.
+
+* Production: https://relay.firefox.com/
+* Stage: https://fx-relay-demo.netlify.app
+
+**NOTE**:
+* Some locales have more files than other locales. This is due to feature availability and avoiding using community resources for features not available in the mid term or not at all.
+* The staging server is not ideal. It syncs once a day. Premium features are not available for testing. Not all free service features can be tested.
+
+### Localizing the strings
+
+Firefox Relay is written in Fluent, or .ftl format. It contains variables and placeables. Sometimes it contains special punctuations or tags. Make sure to preserve them in the localized strings. For better context, many strings are provided with comments, which can be found above the area where you enter translated strings. If you have additional questions:
```

Comment:
uses Fluent for localization, not is written.
Also, "where you enter translated strings", add that this happens in Pontoon.

Same comment as before on writing short, disconnected sentences.